storybookjs/storybook
# Keycodes

Issue:

## What I did
* Refactored the state structure of keycodes.
* appropriately renders "dead" key codes/ alt gr keycodes, ie( `alt` + `J`  no longer renders `alt` + `∆`)
* fix filter search so that it is dynamic and changes based upon the current search shortcut value
* fixed lint warnings

## How to test

- Is this testable with Jest or Chromatic screenshots?
yes but i do need to update the tests to reflect the changes to the data structure
- Does this need a new example in the kitchen sink apps?
no
- Does this need an update to the documentation?
no

If your answer is yes to any of these, please make sure to include it in your PR.
